lighthouse-core/lib/emulation.js

```diff
@@ -41,10 +41,24 @@ const NEXUS5X_USERAGENT = {
     '(KHTML, like Gecko) Chrome/59.0.3033.0 Mobile Safari/537.36'
 };
 
+/**
+ * Adjustments needed for DevTools network throttling to simulate
+ * more realistic network throttling conditions
```

Comment:
Added a link to the crbug (not sure whether to make the analysis doc public yet)